src/Microsoft.Azure.WebJobs.Host/Queues/DispatchProcessorFactory.cs

```diff
@@ -0,0 +1,36 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+namespace Microsoft.Azure.WebJobs.Host.Queues
+{
+    /// <summary>
+    /// factory that only creates a single instance
+    /// </summary>
+    public class DispatchProcessorFactory
```

Comment:
Naming suggestion: how about DispatchQueueFactory/DispatchQueue? Or perhaps ExecutionQueue or InvocationQueue.

Reply:
I believe I tried `DispatchQueue` before and got `CA1711	Rename type name 'DispatchQueue' so that it does not end in 'Queue'.` I cannot name it Shun**Queue** if it does not extend `System.Collections.Queue`